syntax = "proto3";

message GateSession
{
	uint64 session_id = 1;
}

